lib/index.js

```diff
@@ -47,14 +48,16 @@ function displayFavorites(){
   let locations = userFavoritesObj.favoriteLocations;
   locations.forEach(function(e) {
     let name = e["attributes"]["location"];
-    let newDiv = document.createElement("p");
-    let newText = document.createTextNode(`${name}`);
-    newDiv.appendChild(newText);
-    document.getElementById("favoritesContainer").appendChild(newDiv);
+    var favDropDown =  document.getElementById("menuFavs");
+    let newFav = document.createElement("li");
+    let favText = document.createTextNode(`${name}`);
+
+    newFav.appendChild(favText);
+    document.getElementById("menuFavs").appendChild(newFav);
   });
 }
 
-function getUserFavorites(api_key){
+function getUserFavorites(api_key, callback){
```

Comment:
I like the use of a callback function here. It makes it a lot more plastic when it comes to adding code